server/fishtest/schemas.py

```diff
@@ -635,7 +650,7 @@ def flags_must_match(run):
 # about non-validation of runs created with the prior
 # schema.
 
-RUN_VERSION = 13
+RUN_VERSION = 14
```

Comment:
#2297 has already set RUN_VERSION to 14, I wonder if this PR must set 15

Reply:
Fixed and rebased now.